### DISCUSSION TITLE
feat: ✨ list label for repository card

### DISCUSSION THREAD

**Discussion by konhi** [community/community#8707] [Other Feature Feedback, Questions, & Ideas] [answered]
1. ❌ Currently, user has to click on list menu to check if it belongs to any list.

1. ✔ List label allows the user to **instantly** identify to which list repositories belong.

1. 🎨 Here are some visual suggestions!

![github_feature](https://user-images.githubusercontent.com/61631665/145676436-4c5e8b7a-7eff-4b83-b7cf-e54807cff8c0.PNG)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note! Agreed, it's not very clear right now when a repository belongs to a list. I'll add your +1 to our tracking issue for this.
